dash_table/package-info.json

```diff
@@ -53,8 +53,8 @@
     "@babel/preset-react": "^7.0.0",
     "@cypress/webpack-preprocessor": "^2.0.1",
     "@percy-io/percy-storybook": "^2.0.0",
-    "@storybook/cli": "^4.0.2",
-    "@storybook/react": "^4.0.2",
+    "@storybook/cli": "^4.0.7",
+    "@storybook/react": "^4.0.7",
```

Comment:
Having issues running storybook locally, updating fixes the issue -- local dep of react needs to be updated